source/updateCheck.py

```diff
@@ -519,7 +520,10 @@ def onInstallButton(self, evt):
 	def onPostponeButton(self, evt):
 		finalDest=os.path.join(storeUpdatesDir, os.path.basename(self.destPath))
 		try:
-			os.renames(self.destPath, finalDest)
+			# #9825: behavior of os.rename(s) has changed in Python 3.

```

Comment:
Please expand this with how `os.renames` has changed. Most importantly, what was our original intention that isn't met by `os.renames` now?